8.5-java11/Dockerfile

```diff
@@ -45,8 +45,8 @@ COPY context.xml /usr/local/tomcat/conf/context.xml
 COPY logging.properties /usr/local/tomcat/conf/logging.properties
 COPY web-easyauth.xml /usr/local/tomcat/conf/web.xml
 COPY tmp/shared/easyauth/azure.appservice.easyauth.jar /usr/local/tomcat/lib/
-COPY tmp/shared/easyauth/javax.json-api-1.0.jar /usr/local/tomcat/lib/
-COPY tmp/shared/easyauth/javax.json-1.0.4.jar /usr/local/tomcat/lib/
+COPY tmp/shared/appservice/azure.appservice.jar /usr/local/tomcat/lib/
+COPY tmp/shared/tomcat/bin/setenv.bat /usr/local/tomcat/bin/
```

Comment:
Do we need the 'bin' directory in tmp/shared/tomcat? May be easier to manage by having setenv.bat in tmp/shared/tomcat/setenv.bat

Reply:
Plan was to keep the bin and others folders files in a separate folder.

---
In reply to: [263602292](https://github.com/Azure-App-Service/tomcat/pull/22#discussion_r263602292) [](ancestors = 263602292)